### ISSUE TITLE
v1.2.1, no-commit-to-branch, error: unrecognized arguments: .pre-commit-config.yaml

### ISSUE THREAD

**Issue by dongyuzheng** [pre-commit/pre-commit-hooks#268]
Ran `pre-commit autoupdate` updating v1.2.0 to v1.2.1

```
Don't commit to branch...................................................Failed
hookid: no-commit-to-branch

usage: no-commit-to-branch [-h] [-b BRANCH]
no-commit-to-branch: error: unrecognized arguments: .pre-commit-config.yaml
```

**Comment by asottile** [comment]
Indeed, I can reproduce this as well -- it's a bug with the configuration -- I'll get a fix for that!

```console
$ git commit -m 'foo'
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Don't commit to branch...................................................Failed
hookid: no-commit-to-branch

usage: no-commit-to-branch [-h] [-b BRANCH]
no-commit-to-branch: error: unrecognized arguments: .pre-commit-config.yaml
```

**Comment by asottile** [comment]
This is fixed in [v1.2.1-1](https://github.com/pre-commit/pre-commit-hooks/releases/tag/v1.2.1-1)
